sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -464,6 +480,11 @@ public enum HttpPipelinePosition
         PerRetry = 1,
         BeforeTransport = 2,
     }
+    public static partial class MultipartResponse
```

Comment:
Curious: why add this type to the public API as part of this change?

Reply:
I would hesitate to put this type in the public API unless there is a clear use case for it from a customer perspective -- either a library author or an end user customer (typically the former for shared source).  There is usually a way to work around the Core/internal divide in other ways.

If we do decide it makes sense to put this in the public API, I think we should take a step back and get clear on what the requirements/needs for this type are and make sure we design it appropriately.